src/main/scala/NumberToLongShort.scala

```diff
@@ -0,0 +1,48 @@
+
+import scala.util.{Failure, Success, Try}
+
+class NumberToLongShort {
+
+  lazy val english: List[String] = List(" ", " thousand and ", " million ", " billion ", " trillion ", " quadrillion ", " quintillion ", " sextillion ")
+  lazy val french: List[String] = List(" ", " mille et ", " million ", " milliard ", " billion ", " billiard ", " trillion ", " triliard ")
+
+  def checkIfNumber(input: String, language: List[String]): String = {
+    Try {
+      input.toLong
+    } match {
+      case Success(a) => langOutput(input, language)
+      case Failure(a) => "Enter a valid positive number"
+    }
+  }
+
+  def langOutput(input: String, language: List[String]): String = {
+    if (input.contains('-')) {
+      langNeg(input, language)
+    }
+    else {
+      langPos(input, language)
+    }
+  }
+
+  def langPos(input: String, language: List[String]): String = {
+    val in = input.replaceFirst("^0*", "").reverse.grouped(3).toList.map(_.reverse.toList.mkString)
+    val out = calculation(in, language)
+    out.reverse.mkString
```

Comment:
val out = calculation(in, language)
out.reverse.mkString
could just be
calculation(in, language).reverse.mkString
remember that the last statement is returned